app/src/androidTest/java/com/duckduckgo/app/browser/BrowserWebViewClientTest.kt

```diff
@@ -1184,7 +1191,12 @@ class BrowserWebViewClientTest {
         var countFinished = 0
         var countStarted = 0
 
-        override fun onPageStarted(webView: WebView, url: String?, site: Site?) {
```

Comment:
I'm removing the site here because it was only used to check `isDesktopMode`. However, site isn't assigned until after `onPageStarted` is called, therefore it was null for new tabs, and contained old data for every subsequent page load. It still worked for `isDesktopMode`, because if site is null, we default to setting it as false, and every time we switch between desktop and mobile modes a page refresh is triggered, but can lead to confusion and issues for virtually any other site property